<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leaderboard</class>
 <widget class="QWidget" name="leaderboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(112, 112, 112);</string>
  </property>
  <widget class="QLabel" name="player_name">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>30</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
background-color: rgb(0, 0, 0);
border-top-left-radius:15px;</string>
   </property>
   <property name="text">
    <string> PLAYER'S NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>30</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>         SCORE</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>30</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>         TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="leaderBoardName">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>80</y>
     <width>221</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);
color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;
border: 5px solid black;


</string>
   </property>
   <property name="text">
    <string>ahmet</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="seperate_l">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>8</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(68, 68, 68);</string>
   </property>
   <property name="text">
    <string>|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
</string>
   </property>
  </widget>
  <widget class="QLabel" name="seperate_l_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>8</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(68, 68, 68);</string>
   </property>
   <property name="text">
    <string>|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
</string>
   </property>
  </widget>
  <widget class="QLabel" name="leaderBoardScore">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>191</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);
color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;
border: 5px solid black;


</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="leaderBoardTime">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>80</y>
     <width>241</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(68, 68, 68);
color:rgb(255,255,255);
font: 12pt &quot;Algerian&quot;;
border: 5px solid black;


</string>
   </property>
   <property name="text">
    <string>5 sn</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
